config.yaml

```diff
@@ -14,14 +14,15 @@ carpentry: 'lc'
 title: 'Python Intro for Libraries'
 
 # Date the lesson was created (YYYY-MM-DD, this is empty by default)
-created: '2018-04-12'
+created: '2024-05-24'
```

Comment:
Is it your intention to set a new creation date? Is this version effectively a completely new lesson, rather than a modification of what was here before? In my view, the creation date is best aligned with the beginning of the repository commit history...

Reply:
Thanks @tobyhodges! It is a major revision, but not completely new, so I change the created date back to 2018.